src/containers/SignUp/index.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import {View, Text, SafeAreaView} from 'react-native';
+import {Formik} from 'formik';
+import { CheckBoxField, TextInputField, CustomButton } from '../../components';
+
+const SignUp = () => {
+    return (
+        <SafeAreaView>
+            <Formik>
+            <View>
+               <View>
+               <TextInputField label='Nombre'/>
```

Comment:
Codacy has a fix for the issue: [Unexpected usage of singlequote.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
               <TextInputField label="Nombre"/>
```